syntax = "proto3";

package authentication;

service AuthService {
  rpc ValidateToken (TokenRequest) returns (TokenResponse);
}

message TokenRequest {
  string token = 1;
}

message TokenResponse {
  bool valid = 1;
}